app/helpers/search_helper.rb

```diff
@@ -22,16 +22,22 @@ def link_to_result(result)
     end
   end
 
-  def link_to_tab(target)
-    if @active_tab == target.downcase
-      link_to target, results_path(params.permit(:q, :per_page, :booleanType, :tab).merge(tab: target.downcase)),
-        aria: { current: "page" },
-        class: "active tab-link",
-        data: { turbo_frame: "search-results", turbo_action: "advance" }
+  # Generates a link for a search results tab, marking it as active if it matches the current active tab.
+  # @param target [String] The name of the tab to link to
+  # @param label [String] The display label for the tab (optional)
+  # @return [String] HTML link element for the tab
+  def link_to_tab(target, label = nil)
```

Comment:
I wondered how soon we'd need to have a `label` argument in this helper - I guess the answer is "a few days".

My question about this logic, as I follow the interactions between `label` and `target` inside the method, is whether we need tests for this behavior. If I'm following this correctly, we derive `clean_target` for use in the DOM first, and then overlay `label` atop `target`, so that `target` gets used in human-facing contexts like the visible text on the UI. I like that logic and sequence, so I don't have a concern there - but after dealing with UI bugs all day I'm starting to wonder if we need more tests for this type of thing.